tests/test_atlalign.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
```

Comment:
Do you mind adding the License header here too ?

Reply:
Done in 9ad3ee9